src/BrowserDetector.js

```diff
@@ -1,9 +1,6 @@
-import memoize from 'lodash/memoize';
 
-export const isFirefox = memoize(() =>
-  /firefox/i.test(navigator.userAgent)
-);
+const _isSafari = Boolean(window.safari);
+const _isFirefox = !_isSafari && /firefox/i.test(navigator.userAgent);
```

Comment:
That also seems like a positive change to me.

Let's just guard these in checks so that this doesn't blow up server rendering!